class-factory/native/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "class-factory"
+version = "0.1.0"
+build = "build.rs"
+exclude = ["artifacts.json", "index.node"]
+
+[lib]
+name = "class_factory"
+crate-type = ["cdylib"]
```

Comment:
Should we change all the `dylib`'s in the examples to `cdylibs`?

Reply:
Yes, it's a fix in the template that is part of `0.3.1`.

Reply:
Nice! Should we make those changes to the other examples in the repo in another PR or in this PR?

Reply:
Actually I'll do this in a separate PR